tests/examples/opinions/united_states/tax_example.compare.json

```diff
@@ -2,7 +2,7 @@
   {
     "case_dates": "2021-12-08",
     "case_names": "Alice J. Coggin",
-    "download_urls": "https://public-api-blue.dawson.ustaxcourt.gov/public-api/21580-19/919cb4ce-fb26-4cb3-a0f3-a1a2f0e6927b/public-document-download-url",
+    "download_urls": "https://public-api-green.dawson.ustaxcourt.gov/public-api/21580-19/919cb4ce-fb26-4cb3-a0f3-a1a2f0e6927b/public-document-download-url",
```

Comment:
Hm, this looks like they're doing [green/blue deployments](https://docs.aws.amazon.com/whitepapers/latest/overview-deployment-options/bluegreen-deployments.html), and makes me wonder if it'll swap back to blue next time they release code. I hope not, but it's possible. I guess we'll see.

Reply:
I saw the blue green deployment, and assumed that blue was from all the test and green was the actual production.  When we built this, it seemed very much to be in development

Reply:
We shall see!